anchore_engine/services/apiext/swagger/swagger.yaml

```diff
@@ -2970,9 +3227,6 @@ definitions:
         type: string
       trigger_id:
         type: string
-      expires_on:
```

Comment:
Yeah, don't want to remove this. It's part of another feature...just highlighting this so we for sure do not merge :)